rosetta-lang/src/main/java/com/regnosys/rosetta/RosettaEcoreUtil.xtend

```diff
@@ -132,7 +132,7 @@ class RosettaEcoreUtil {
 	}
 	
 	def boolean hasMetaDataAnnotations(RAttribute attribute) {
-		attribute.metaAnnotations.exists[name == "reference" || name == "location" || name == "scheme" || name == "id"]
+		attribute.RMetaAnnotatedType.getMetaAttributes.exists[name == "reference" || name == "location" || name == "scheme" || name == "id"]
 	}
```

Comment:
I wonder if we still need this method? Perhaps it's a good idea reviewing its usage.